### PR TITLE
Add filemode attribute to the DeviceNode definition

### PR DESCRIPTION
**Pull request by bart0sh**
Fixes: #29

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,8 +32,8 @@ $ cat > /etc/cdi/vendor.json <<EOF
       "name": "myDevice",
       "ociEdits": {
         "deviceNodes": [
-          {"path": "/dev/card1", type": "c", "major": 25, "minor": 25, "permissions": "rw", "uid": 1000, "gid": 1000}
-          {"path": "/dev/card-render1", "type": "c", "major": 25, "minor": 25, "permissions": "rwm", "uid": 1000, "gid": 1000}
+          {"path": "/dev/card1", type": "c", "major": 25, "minor": 25, "filemode": 384, "permissions": "rw", "uid": 1000, "gid": 1000}
```
Comment (mikebrow) [inline README.md:35]: fileMode?

File: SPEC.md [hunk 2]
```diff
@@ -98,6 +98,8 @@ The key words "must", "must not", "required", "shall", "shall not", "should", "s
                     "type": "<type>" (optional),
                     "major": <int32> (optional),
                     "minor": <int32> (optional),
+                    // file mode for the device
+                    "filemode": <int> (optional),
```
Comment (mikebrow) [inline SPEC.md:102]: fileMode?

**Review by mikebrow — COMMENTED** [review]
see questions on sp and case

File: SPEC.md [hunk 3]
```diff
@@ -180,6 +182,7 @@ The `containerEdits` field has the following definition:
     * `type` (string, OPTIONAL) Device type: block, char, etc.
     * `major` (int64, OPTIONAL) Device major number.
     * `minor` (int64, OPTIONAL) Device minor number.
+    * `filemod` (int64, OPTIONAL) file mode for the device.
```
[the remark this answers is not in the document]
Comment (bart0sh) [reply SPEC.md]: fixed. Thank you for the review!

**Review by mikebrow — APPROVED** [review]
LGTM

**Comment by elezar** [comment]
Thanks @bart0sh. This looks good. I was out, but it seems as if you were able to get it reviewed while I was gone.

**Pull request merged**